crates/bevy_gltf/src/loader.rs

```diff
@@ -263,6 +311,24 @@ async fn load_gltf<'a, 'b>(
         if let Some(Err(err)) = err {
             return Err(err);
         }
+
+        for (&entity, &skin_index) in &entity_to_skin_index_map {
```

Comment:
So it looks like this grabs animation data from the .gltf file, creating one entity for each joint?

Reply:
Technically, it's only skin data, without any animation. Entity joints are already created since they are part of the glTF nodes. This code just map the created entity to the joint defined in glTF.